drivers/storage/isilon/executor/isilon_executor.go

```diff
@@ -43,6 +44,18 @@ func (d *driver) Name() string {
 	return isilon.Name
 }
 
+// Supported returns a flag indicating whether or not the platform
+// implementing the executor is valid for the host on which the executor
+// resides.
+func Supported(ctx types.Context, opts types.Store) (bool, error) {
+	// make sure mount.nfs is present
+	if err := exec.Command(
```

Comment:
Oh, nice! `LookPath()` is very convenient. It's `which` in Go. :)

I'm happy to change it. Do you prefer me to use `LookPath()` directly, or to import `gotil` ?